db/data_migration/20190812093415_change_tribunal_org_type.rb

```diff
@@ -0,0 +1,9 @@
+orgs = Organisation.where(organisation_type_key: :tribunal_ndpb)
+
+orgs.each do |org|
+  puts "Changing #{org.name} old org type #{org.organisation_type_key}"
+
+  org.update_attributes!(organisation_type_key: :tribunal)
```

Comment:
`tribunal` isn't a value listed here https://github.com/alphagov/whitehall/blob/master/app/models/organisation_type.rb#L2-L18

Does that matter?

Reply:
There's another PR https://github.com/alphagov/whitehall/pull/4969/files